docs/guides/akamai/get-started/observability-with-datastream-and-trafficpeak/index.md

```diff
@@ -0,0 +1,93 @@
+---
+slug: observability-with-datastream-and-trafficpeak
+title: "Large Data Observability With DataStream and Hydrolix TrafficPeak"
+description: "This guide reviews Akamai's managed observability solution, Hydrolix TrafficPeak, including product features, how TrafficPeak overcomes observability challenges, and a proven implementation architecture."
+authors: ["John Dutton"]
+contributors: ["John Dutton"]
+published: 2024-06-11
+keywords: ['observability','datastream','trafficpeak','hydrolix','logging','data logging','visualization']
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+external_resources:
+- '[Official Hydrolix Documentation](https://docs.hydrolix.io/docs/welcome)'
+- '[Official TrafficPeak Site](https://hydrolix.io/partner-program/trafficpeak/)'
+---
+
+## Overview
+
+Observability workflows are critical to gaining meaningful insight to your application’s health, customer traffic, and overall performance. However, there are challenges that come along with achieving true observability, including large volumes of traffic data, data retention, time to implementation, and the cost of each.
```

Comment:
@jddocs I'm not quite sure what is being referred to at the end with "the cost of each" (i.e. the cost of each <what>)

Reply:
It was referring to everything after "including", so it's referencing: large volumes of traffic data, data retention, and time to implementation